app/projects.server.ts

```diff
@@ -1,11 +1,18 @@
 import projects from "./data/projects.json";
 import type { Project, ProjectCategory } from "./types";
 
+/**
+ * Returns a list of projects
+ */
 export function getProjects() {
   return projects as Array<Project>;
 }
 
-export async function getProject(slug: string): Promise<Project | undefined> {
+/**
+ * Returns a project that matches the slug provided, or `undefined` if there's
+ * no match.
+ */
+export function getProject(slug: string): Project | undefined {
```

Comment:
This method didn't need to be asynchronous. I updated its implementation in `app/routes/$projectOwner/$project.tsx`.